pkg/kubelet/kubelet.go

```diff
@@ -91,6 +95,7 @@ type volumeMap map[string]volume.Interface
 // New creates a new Kubelet for use in main
 func NewMainKubelet(
 	hostname string,
+	containerRuntimeChoice string,
```

Comment:
re: line 121/126 (GH won't let me comment down there) - we don't need to wait for Docker if containerRuntimeChoice is rocket, do we?